qiita_pet/test/test_study_handlers.py

```diff
@@ -686,8 +686,17 @@ def test_post_admin(self):
         self.assertEqual(Artifact(3).visibility, 'sandbox')
 
 
-@qiita_test_checker()
 class TestAddArtifact(TestHandlerBase):
+    database = True
```

Comment:
It looks weird, but `database` is a class attribute from `TestHandlerBase` and it is used in the `setUp` from that class to decide whether we need to reset the DB or not after each test.